Docs/Book/RDKit_Book.rst

```diff
@@ -2701,6 +2701,206 @@ path) enumeration algorithm used in the RDKit fingerprint. After a subgraph has
 been generated, it is used to set multiple bits based on different atom and bond
 type definitions.
 
+Self-Contained Structure Representations (SCSR) for Macromolecules
+*******************
+
+The SCSR support tentative added to RDKit follows the description in the BIOVIA document “biovia_ctfileformats_2020.pdf” available from 
+“CTfile Formats - Dassault Systèmes”.  That document does not provide any real detail for that format, and contains one example file.
+
+In addition, Biovia Draw supports reading and writing this format.  As much as possible, the RDKit support allows the functionality
+supported by Biovia Draw.  One exception is the RDkit treatment of hydrogen bonds in SCSR files/blocks (vide infra).
+
+
+Representation
+==============
+
+Main mol
+--------
+An SCSR file contains a mol with a CTAB in v3000 format.   That CTAB can contain elemental atoms and macro atoms corresponding to amino acids (AA), RNA, and DNA elements.  
+The RNA and DNA elements are represented by three parts – a phosphate group, a sugar, and a base.  
+
+Each atom line in the CAT can refer to an elemental atom or a macro atom.  Macro atom lines have a text description of the macro name and must have a CLASS and an
+ATTCHORD attributed.  They can also have an optional SEQID attribute.  
+
+According to the Biovia doc, the CLASS attribute must have a value that is one of:  AA, dAA, ScsrSupportInRDKitDNA, RNA, SUGAR, BASE, PHOSPHATE, LINKER, CHEM, LGR,P MODAA, MODdAA, MODDNA, MODRNA, XLINKAA, XLINKdAA, XLINKDNA, XLINKRNA.   
+This implementation in RDKit does NOT enforce the allowed list.
```

Comment:
I think it does enforce the allowed list.

Reply:
SO I took a look.  The class in an atom definition is NOT checked.  There is an example in Code/GraphMol/FileParsers/file_parsers_catch.cpp that has a class of "foo".

The class in an SGROUP is checked.   In SCSR block parsing, the atom line will have a class which is not checked against the allowed list, but this must also appear in an SGROUP and that is checked, so the SCSR treatment DOES enforce the allowed list.  

Thanks for catching this.  I have changed the doc to reflect this.